pkgs/tools/misc/toybox/default.nix

```diff
@@ -0,0 +1,67 @@
+{
+  stdenv, lib, fetchFromGitHub, buildPackages,
+  enableStatic ? false,
+  enableMininal ? false,
+  extraConfig ? ""
+}:
+stdenv.mkDerivation rec {
+  name = "toybox-${version}";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "landley";
+    repo = "toybox";
+    rev = "b11e1206b4193dd46b4c40c333175fceadd409df";
```

Comment:
Got it :-). Used the next revision since the release version indentifies as 0.7.8 instead of 0.8.0; see landley/toybox#114.

If it's better to use the release commit, I can do that instead :-).

Reply:
Updated to use the release commit.